pkg/health/server/server.go

```diff
@@ -191,87 +191,71 @@ func (s *Server) collectNodeConnectivityMetrics() {
 			continue
 		}
 
-		targetClusterName, targetNodeName := getClusterNodeName(n.Name)
 		nodePathPrimaryAddress := healthClientPkg.GetHostPrimaryAddress(n)
 		nodePathSecondaryAddress := healthClientPkg.GetHostSecondaryAddresses(n)
 
 		endpointPathStatus := n.HealthEndpoint
 		isEndpointReachable := healthClientPkg.SummarizePathConnectivityStatusType(healthClientPkg.GetAllEndpointAddresses(n)) == healthClientPkg.ConnStatusReachable
 		isNodeReachable := healthClientPkg.SummarizePathConnectivityStatusType(healthClientPkg.GetAllHostAddresses(n)) == healthClientPkg.ConnStatusReachable
 
-		location := metrics.LabelLocationLocalNode
-		if targetClusterName != localClusterName {
-			location = metrics.LabelLocationRemoteInterCluster
-		} else if targetNodeName != localNodeName {
-			location = metrics.LabelLocationRemoteIntraCluster
-		}
-
 		// Aggregated status for endpoint connectivity
 		metrics.NodeConnectivityStatus.WithLabelValues(
-			localClusterName, localNodeName, targetClusterName, targetNodeName, location, metrics.LabelPeerEndpoint).
+			localClusterName, localNodeName, metrics.LabelPeerEndpoint).
 			Set(metrics.BoolToFloat64(isEndpointReachable))
 
 		// Aggregated status for node connectivity
 		metrics.NodeConnectivityStatus.WithLabelValues(
-			localClusterName, localNodeName, targetClusterName, targetNodeName, location, metrics.LabelPeerNode).
+			localClusterName, localNodeName, metrics.LabelPeerNode).
 			Set(metrics.BoolToFloat64(isNodeReachable))
```

Comment:
This is setting a value of metric to 0 or 1. We probably want to set it to sum of all reachable nodes.
I would consider adding a new label, something like "outcome" with values "reachable"/"unreachable" and setting value to sum of all reachable and unreachable nodes respectively.

Reply:
Updated to reflect this now